sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/RequestResponseChannel.java

```diff
@@ -91,18 +91,25 @@
         this.operationTimeout = retryOptions.getTryTimeout();
         this.retryPolicy = RetryUtil.getRetryPolicy(retryOptions);
 
+        // Create sender
         this.replyTo = entityPath.replace("$", "") + "-client-reply-to";
         this.messageSerializer = messageSerializer;
+        session.setOutgoingWindow(Integer.MAX_VALUE);
```

Comment:
This is used by Event Hubs too. Why are we setting You should be creating PRs so we can review these changes.

session.open() has to be invoked on the reactor dispatcher thread.